app/models/work.rb

```diff
@@ -14,12 +14,8 @@ class Work < ApplicationRecord
   has_many :attached_files, dependent: :destroy
   has_many :keywords, dependent: :destroy
 
-  validates :abstract, :access, :state, :title, presence: true
-  validates :contact_email, presence: true, format: { with: Devise.email_regexp }
+  validates :state, presence: true
```

Comment:
I think we still want the work_type and work_subtype validators here.  I think we also want all the format validators here (just not the presence validations)

Reply:
ok